xt/41-coaload.t

```diff
@@ -1,6 +1,7 @@
 use Test::More;
 use strict;
 use warnings;
+use LedgerSMB::Sysconfig;
```

Comment:
@ehuelsmann actually, the required PGHOST and PGPORT variables aren't documented there.
and as we've just been discussing in #ledgersmb there was no default value applied to either of those env vars which broke running with no ledgersmb.conf file

I'd say that #2635 will supersede this PR as it correctly sets defaults from and the env-vars from ledgersmb.conf

#2635 doesn't however add PGHOST and PGPORT to  https://github.com/ledgersmb/LedgerSMB/tree/master/t#environment-variables